test/rekt/resources/sequence/sequence.go

```diff
@@ -127,7 +127,7 @@ func WithStepFromDestination(dest *duckv1.Destination) manifest.CfgFn {
 		if dest.CACerts != nil {
 			// This is a multi-line string and should be indented accordingly.
 			// Replace "new line" with "new line + spaces".
-			step["CACerts"] = strings.ReplaceAll(*dest.CACerts, "\n", "\n      ")
+			step["CACerts"] = strings.ReplaceAll(*dest.CACerts, "\n", "\n       ")
```

Comment:
Getting `sequence.go:56: error converting YAML to JSON: yaml: line 42: did not find expected key` in the auth tests. Seems we are missing correct indentation here.
IIRC you need to indent it by 2 spaces (like in test/rekt/resources/delivery/delivery.go):
```suggestion
			step["CACerts"] = strings.ReplaceAll(*dest.CACerts, "\n", "\n        ")
```